.github/ISSUE_TEMPLATE/2_bug-or-performance-issue-report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug or performance issue report
+about: Create a report to help us improve oneDPL
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug or performance issue is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+The following information might be useful: 
+ - CMake command: [e.g.cmake -DCMAKE_CXX_COMPILER=icpx -DCMAKE_BUILD_TYPE=release -DONEDPL_PAR_BACKEND=tbb]
```

Comment:
Please use "for example" instead of "e.g." in all instances.

Reply:
Done.